analytic_mixin_analytic_account/models/analytic_mixin.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Quartile Limited (https://www.quartile.co)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class AnalyticMixin(models.AbstractModel):
+    _inherit = "analytic.mixin"
+
+    analytic_account_ids = fields.Many2many(
+        "account.analytic.account",
+        compute="_compute_analytic_account_ids",
+        string="Analytic Accounts",
+        help="Analytic accounts computed from analytic distribution.",
+    )
+    analytic_account_names = fields.Char(
+        compute="_compute_analytic_account_ids",
+        help="Comma-separated analytic account names, in case it is useful to be "
+        "included in the exported data.",
+    )
+
+    def _compute_analytic_account_ids(self):
+        analytic_account = self.env["account.analytic.account"]
```

Comment:
```suggestion
        AnalyticAccount = self.env["account.analytic.account"]
```

or `analytic_account_model`

Reply:
I update the code to use as `analytic_account_model`.